examples/reactnative/rn_0.59/README.md

```diff
@@ -1,46 +0,0 @@
-# React Native
```

Comment:
It'd be worth copying this over to the 0.63 project and adapting where necessary

Reply:
Absolutely.  I actually meant to and forgot.